terraform/terraform_spec.rb

```diff
@@ -23,6 +23,14 @@
     expect(file("/etc/alpine-release")).to be_file
   end
 
+  it "installs the right version of Alpine" do
+    expect(os_version).to include("Alpine Linux 3.3")
+  end
```

Comment:
I'm not sure what's the value of pinning the version and running tests that do not check it?
If we want to be sure that terraform is in specific version as it contains fixes that we want then we should check if this specific version was installed. And in my opinion the same applies to any other software that we use including OS.